cms/standard_pages/models.py

```diff
@@ -38,3 +41,92 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    summary = RichTextField(features=settings.RICH_TEXT_BASIC)
+    featured_items = StreamField(
+        [("featured_item", RelatedContentBlock())],
+        help_text=_("Leave blank to automatically populate with child pages. Only published pages will be displayed."),
+        blank=True,
+    )
+
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        FieldPanel("featured_items"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("summary"),
+        index.SearchField("content"),
+    ]
+
+    def get_formatted_items(self, request: "HttpRequest") -> list[dict[str, str]]:
+        """Returns a formatted list of Featured items
+        that can be either children internal Pages or specified in a RelatedContentBlock
+        for use with the Design system Document list component.
+        """
+        formatted_items = []
+        if featured_items := self.featured_items:
+            for featured_item in featured_items:
```

Comment:
Nit: The `if` and `for` here can be combined. `featured_items` will be an empty list if unset. The below `else` can then become `if not formatted_items:`. This removes a bit of indentation.

Reply:
[2dfb6f4](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/2dfb6f4fb80bab1e5917fe81e27b1f1a55394914)